internal/service/platform/connector/k8s_rancher_data_source.go

```diff
@@ -0,0 +1,46 @@
+package connector
+
+import (
+	"github.com/harness/terraform-provider-harness/helpers"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DatasourceConnectorRancher() *schema.Resource {
+	resource := &schema.Resource{
+		Description: "Datasource for looking up a Rancher connector.",
```

Comment:
```suggestion
		Description: "Data source for looking up a Rancher connector.",
```
(Maybe check globally?)

Reply:
Done